config.yaml

```diff
@@ -119,6 +119,8 @@ languages:
               link: https://numpy.org/doc/stable
             - text: Learn
               link: /learn
+            - text: Citing Numpy
+              link: /citing-numpy
```

Comment:
I thought you wanted this in the footer? This adds it to the top navbar, which seems a little too prominent.

Reply:
LGTM.:) Here is a screenshot of the build.
<img width="1437" alt="npPR407_footer" src="https://user-images.githubusercontent.com/43481325/111101954-b26cde80-8596-11eb-81d8-a3a9cbaf431b.png">

Reply:
Oops I misread the diff, you're completely right - sorry for the noise!